CHANGELOG.md

```diff
@@ -9,12 +9,18 @@
 - refactor(db): remove artifacts as well as old init scripts and restructure directories ([#388])
 - feat: disable size limit for Mapping Saturation ([#498])
 - Overhaul docker compose setup ([#483])
+- remove JRC related layers and report `jrc_requirements` ([#503])
+
+### How to Upgrade
+
+
```

Comment:
I think its not necessary to add this change to `how to upgrade` section. But I would move this from the `Other Changes` to `Breaking Changes` section.

Reply:
done